### DISCUSSION TITLE
Looking for recommandations on my ssh tmux &| tee workflow

### DISCUSSION THREAD

**Discussion by stephane-archer** [fish-shell/fish-shell#9259] [Q&A] [answered]
Hi, I found myself connecting to remote servers using `ssh` and `tmux` (remotely) and then running
```
./MyScript.fish &| tee MyLogFile.txt
```
So I can quickly review what is going on and If something was unexpected, have a look at the logs, because I can't sometimes scroll to the beginning of the issue with `tmux` and I can use grep and other UNIX tools.

Reading that I was wondering if you knew a better solution to do what I do.

**Comment by faho** [comment, marked as answer]
(migrated this to discussions because it's a better fit)

In general, my position on things like this is: Do whatever works for you. I'm not your dad, I'm not gonna tell you to do it my way or the highway.

I'm also a friend of a low-abstraction kind of workflow, so I wouldn't recommend a function that runs `&| tee MyLogFile.txt`. That's just one more thing to forget. Stick to the basics.

So I really don't have a recommendation to make here. It's fine. Just do that. It's okay. You have my blessing.

Or you could look into why you can't scroll back in tmux, but that's likely to be a tmux question, not a fish one.

**Reply by faho** [reply]
@stephane-archer I assume that answers your question.

**Reply by stephane-archer** [reply]
It's a great answer, sometimes we do things because we don't know a better way. If people can mention what they do here, i will be glad to hear what they use. :)
